pages/Documents/ConversationalAI/IntentManager/key-terms-concepts.md

```diff
@@ -145,34 +145,48 @@ A meta intent is a wrapper that can contain many other standard intents. This fu
 
 ### Entities
 
-Entities are keywords or expressions that represent groups of items. For example, an entity named `SIZE` might represent the following values: small, medium, and large.
+Entities are the things, and their attributes, referred to in consumer conversations. More formally, a named entity is a set of elements that is important to understanding raw text, but is not necessarily related to the *intent* of the raw text. In general, entities can be swapped while preserving the general intent of the text.
```

Comment:
Is there any word we can use to describe entities other than "the things"? Even the previous description of "keywords or expressions" sounds better.